ContactDetailsApi/V1/Boundary/Request/ContactQueryParameter.cs

```diff
@@ -8,6 +8,9 @@ public class ContactQueryParameter
         [FromQuery(Name = "targetId")]
         public Guid? TargetId { get; set; }
 
+        [FromQuery(Name = "id")]
+        public Guid? Id { get; set; }
+
```

Comment:
You should create a new request object for this call rather than co-opting an existing one. That way you can also add the necessary validation without affecting existing functionality.